statsmodels/tsa/arima_process.py

```diff
@@ -311,36 +291,70 @@ def arma_impulse_response(ar, ma, nobs=100):
 
     MA(2)
 
-    >>> arma_impulse_response([1.0], [1., 0.5, 0.2], nobs=10)
+    >>> arma_impulse_response([1.0], [1., 0.5, 0.2], leads=10)
     array([ 1. ,  0.5,  0.2,  0. ,  0. ,  0. ,  0. ,  0. ,  0. ,  0. ])
 
     ARMA(1,2)
 
-    >>> arma_impulse_response([1.0, -0.8], [1., 0.5, 0.2], nobs=10)
+    >>> arma_impulse_response([1.0, -0.8], [1., 0.5, 0.2], leads=10)
     array([ 1.        ,  1.3       ,  1.24      ,  0.992     ,  0.7936    ,
             0.63488   ,  0.507904  ,  0.4063232 ,  0.32505856,  0.26004685])
-    '''
-    impulse = np.zeros(nobs)
+    """
+    if 'nobs' in kwargs:
+        leads = kwargs['nobs']
+        import warnings
+        warnings.warn(DeprecationWarning, 'nobs is deprecated in favor of leads')
+    impulse = np.zeros(leads)
     impulse[0] = 1.
     return signal.lfilter(ma, ar, impulse)
 
 
-#alias, easier to remember
-arma2ma = arma_impulse_response
+def arma2ma(ar, ma, lags=100, **kwargs):
+    """
+    Get the MA representation of an ARMA process
```

Comment:
Here I mean approximation in the sense of "not an analytic solution".  Other docstrings are explicit about analytic vs approximation.